scripting/pugsetup/mapvote.sp

```diff
@@ -1,9 +1,53 @@
 #define RANDOM_MAP_VOTE "-1"  // must be in invalid index for array indexing
 
+char mapGroups[][] = {
```

Comment:
As mentioned previously, I disagree with placing these in a different directory.

I also disagree with  breaking backwards compatibility for existing users populating a different map config file.